src/core/fetchers/manifest/manifest_fetcher.ts

```diff
@@ -722,7 +735,10 @@ export interface IManifestFetcherSettings {
    * `undefined` will lead to a default, large, timeout being used.
    */
   requestTimeout : number | undefined;
+
+  connectionTimeout: number | undefined;
   /** Limit the frequency of Manifest updates. */
+
```

Comment:
Why a space in between the property and its documentation? I was afraid that it removed the ability to let coding tools make the link between those but it still does it apparently.
Still, visually it does make a separation